stores/localeStore.js

```diff
@@ -0,0 +1,21 @@
+import { defineStore } from 'pinia'
+
+export const useLocaleStore = defineStore('locale', {
+    state: () => ({
+        locale: { lable: 'English', code: 'en' }
+    }),
+    getters: {
+        currentLocale: state => state.locale
+    },
+    actions: {
+        changeLocale() {
+            if (this.currentLocale.code === 'en') {
+                this.locale.lable = '日本語'
```

Comment:
After updating to typescript, it would be better to make this more object oriented and switch the entire locale at once rather than updating every field within. E.g. perhaps you would create a "type", "const", "enum" perhaps even, to represent `en` and `ja` locales

Reply:
Thank you for the advice! I will work on it :)

Reply:
Hi @ann-kilzer, I just made some changes to the file by changing it to TypeScript. Is this the correct way to do it? :)